### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Mesi21**
- Deleted all the boilerplate from CRA (text, images, styles).
- Added the components folder.
- Inside components - Calculator.js fille
- Developed just the UI of the calculator

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,10 @@
...
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div id="app">
+    <Calculator />
+  </div>
+);
```
Comment (Ghiftee) [inline src/App.js]: - [x] Please use class-based components. e.g
```javascript
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import '../index.css';
+
+function Calculator() {
+  return (
+    <div id="app">
+      <Display />
+      <ButtonPanel />
+    </div>
+  );
+}
```
Comment (Ghiftee) [inline src/components/Calculator.js]: - [x] Please use class-based components. e.g
```javascript

class Calculator extends Component {
  render() {
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

```

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @Mesi21 

Good job working on the project so far! 
✔️ Calculator component matches the given design.
✔️ The app runs smoothly.
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by dasileker — APPROVED** [review]
## **Hi @Mesi21,** 

### **Nice Work 👏💯**

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**